frontend/database/local-storage.ts

```diff
@@ -204,7 +192,7 @@ export class LocalStorageDatabase extends IWalletDatabase {
 
     // If does not exist it means that this is a brand new account
     if (!exist) {
-      this._setTokens(defaultTokens);
+      this._setTokens([...defaultTokens]);
```

Comment:
`_setTokens` was throwing an error when trying to run `.sort()` on passed in `defaultTokens` array.